less/quicksilver/jakarta/template.less

```diff
@@ -89,4 +89,68 @@ main {
   .img-responsive{
     display:inherit;
   }
+}
+
+.download-highlights-item {
+  padding:0 30px;
+  @media (max-width: @screen-sm-max) {
+    padding:30px;
+  }
+}
+
+.jakarta-download-delivers {
+  text-align:center;
+  padding:30px 0 40px 0;
+}
+
+.jakarta-download-community {
+  background-image:url("/images/download/community.jpg");
+  background-repeat:no-repeat;
+  background-size:cover;
+  text-align:center;
+  color:@white;
+  padding:30px 0;
+  h2 {
+    color:@white;
+  }
+}
+
+.header-download-bg-img {
+  background-image:url("/images/download/header.jpg");
+  background-repeat:no-repeat;
+  background-size:cover;
+}
+
+.heading-underline {
+  text-transform:none;
+}
+
+.jakarta-download-participation {
+  background-image:url("/images/download/donate.jpg");
+  background-repeat:no-repeat;
+  background-size:cover;
+  padding:40px 0;
+  .featured-donate-icon {
+    background-color: #fff;
+    border-radius: 200px;
+    height: 300px;
+    width: 300px;
+    display: block;
+    padding: 40px;
+    padding-left: 58px;
+    @media (max-width: @screen-sm-max) {
```

Comment:
We should probably flip this so that we do mobile-first enhancement, rather than forcing mobile to work in a desktop environment